src/sentry/static/sentry/app/views/projectsDashboard/deploys.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {Project, Deploy as DeployType} from 'app/types';
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+import SentryTypes from 'app/sentryTypes';
+import TextOverflow from 'app/components/textOverflow';
+import TimeSince from 'app/components/timeSince';
+import Version from 'app/components/version';
+import space from 'app/styles/space';
+import getDynamicText from 'app/utils/getDynamicText';
+
+const DEPLOY_COUNT = 2;
+
+type Props = {
+  project: Project;
+};
+
+const Deploys = ({project}: Props) => {
+  const flattenedDeploys = Object.entries(project.latestDeploys || {}).map(
+    ([environment, value]): Pick<
+      DeployType,
+      'version' | 'dateFinished' | 'environment'
+    > => ({environment, ...value})
+  );
+
+  const deploys = (flattenedDeploys || [])
+    .sort(
+      (a, b) => new Date(b.dateFinished).getTime() - new Date(a.dateFinished).getTime()
+    )
+    .slice(0, DEPLOY_COUNT);
+
+  if (!deploys.length) {
+    return <NoDeploys />;
+  }
+
+  return (
+    <DeployContainer>
+      {deploys.map(deploy => (
+        <Deploy
+          key={`${deploy.environment}-${deploy.version}`}
+          deploy={deploy}
+          project={project}
+        />
+      ))}
+    </DeployContainer>
+  );
+};
+
+Deploys.propTypes = {
+  project: SentryTypes.Project.isRequired,
+};
+
+export default Deploys;
+
+type DeployProps = Omit<Props, 'organization'> & {
```

Comment:
`Props` doesn't have `organization` anymore, do you still need this Omit?

Reply:
I didn't know you could omit things that didn't exist, interesting!